README.md

```diff
@@ -94,6 +98,42 @@ To exclude `node_modules`, see the `exclude` option in the `loaders` config as d
 
 You can also speed up babel-loader by as much as 2x by using the `cacheDirectory` option. This will cache transformations to the filesystem.
 
+### Some files in my node_modules are not transpiled for IE 11
+
+Although we typically recommend not compiling `node_modules`, you may need to when using libraries that do not support IE 11.
+to compile them for IE 11 compatibility.
+
+For this, we suggest passing an `exclude` function for maintainability. You can also use negative lookahead regex as suggested [here](https://github.com/webpack/webpack/issues/2031#issuecomment-294706065).
+
+```javascript
+{
+    test: /\.m?js$/,
+    exclude: function excludeBabelLoader(modulePath) {
+      if (/node_modules/.test(modulePath)) {
+        if (
+          /node_modules[\\/]d3-array[\\/]/.test(modulePath) // compile d3-array to IE 11
+          || /node_modules[\\/](unfetch|d3-scale)[\\/]/.test(modulePath) // compile unfetch and d3-scale to IE 11
+          || /node_modules[\\/]@hapi[\\/]joi-date[\\/]/.test(modulePath) // compile @hapi/joi-date to IE 11
+        ) {
+          return false;
+        }
+        // Exclude other libraries in node_modules
+        return true;
+      }
+      // Compile every others not in node_modules
+      return false;
+    },
```

Comment:
```suggestion
    exclude: {
      test: /node_modules/, // Exclude libraries in node_modules ...
      not: [
        // Except for a few of them that needs to be transpiled because use modern syntax
        /unfetch/,
        /d3-array|d3-scale/,
        /@hapi[\\/]joi-date/,
      ]
    },
```

This is maybe more reasable?

Reply:
Definitely more readable. Maybe lead with this, and mention for more advanced situations use exclude as a function (link to webpack exclude docs)?